tests/unittest/CMakeLists.txt

```
@@ -43,3 +43,13 @@ target_include_directories(unittest PUBLIC ${VDT_INCLUDE_DIR})
       TEST_PREFIX "UT_" # make it possible to filter easily with -R ^UT
       TEST_SPEC ${filter_tests} # discover only tests that are known to not fail
   )
+
+add_executable(algorithms algorithms.cpp)
+target_link_libraries(algorithms PUBLIC FCCAnalyses gfortran PRIVATE Catch2::Catch2WithMain)
+target_include_directories(algorithms PUBLIC ${VDT_INCLUDE_DIR})
+  include(Catch)
+  catch_discover_tests(algorithms
```

```suggestion
add_executable(unittest_algorithms algorithms.cpp)
target_link_libraries(unittest_algorithms PUBLIC FCCAnalyses gfortran PRIVATE Catch2::Catch2WithMain)
target_include_directories(unittest_algorithms PUBLIC ${VDT_INCLUDE_DIR})
  include(Catch)
  catch_discover_tests(unittest_algorithms
```
Building an executable just called `algorithms` is maybe confusing, bettert to prefix with `unittest_`

---

That's probably less important, since it is in the unittest subdir